libs/vm_monitor/src/cgroup.rs

```diff
@@ -115,12 +117,27 @@ impl CgroupWatcher {
             // equal to the logging interval, we can just log the entire buffer every time we set
             // the last entry, which also means that for this log line, we can ignore that it's a
             // ring buffer (because all the entries are in order of increasing time).
+            //
+            // We skip logging the data if data hasn't meaningfully changed in a while, unless
+            // we've already ignored too many (=history_log_len) previous iterations.
             if i == history_log_len - 1 {
-                info!(
-                    history = ?MemoryStatus::debug_slice(&history_log_buf),
-                    summary = ?summary,
-                    "Recent cgroup memory statistics history"
-                );
+                if skipped > history_log_len
```

Comment:
to clarify, now `history_log_len` is both:

1. the number of items that get logged in a single report, and
2. the maximum number of reports that can be skipped

?